Sources/swift-build/usage.swift

```diff
@@ -26,6 +26,8 @@ func usage(_ print: (String) -> Void = { print($0) }) {
     print("  --fetch                        Fetch package dependencies")
     print("  --update                       Update package dependencies")
     print("  --generate-xcodeproj[=<path>]  Generates an Xcode project [-X]")
+    print("  --show-dependencies[=<mode>]   Print dependency graph (text|dot|json)")
+    print("  --show-package                 Print the contents of Package.swift")
```

Comment:
Haha I originally called it `--dump-package`, but then changed it because I thought we were avoiding the word `dump`.